packages/framework/src/Model/Product/Parameter/ParameterRepository.php

```diff
@@ -249,7 +249,9 @@ protected function getProductParameterValuesByProductQueryBuilder(Product $produ
         return $this->em->createQueryBuilder()
             ->select('ppv')
             ->from(ProductParameterValue::class, 'ppv')
+            ->join(ParameterGroup::class, 'pg')
```

Comment:
I have two thoughts about this.
1. The `getProductParameterValuesByProductQueryBuilder` method is overridden in project-base without calling the parent method. This change probably doesn't change any behavior at all. – I suggest moving the method from project-base here and making changes here.
2. ParameterGroup does not have any relation with ProductParameterValue, so I'm not sure how this will behave.